### DISCUSSION TITLE
feature request : virtual pointer

### DISCUSSION THREAD

**Discussion by emnachouchane** [hakimel/reveal.js#3404] [Ideas]
**Use case:** As presenting the slides, I often need to call attention to particular sections of my presentation by highlighting them.Yet, this may be hard to do with a conventional mouse pointer, particularly when demonstrating complicated information as well as diagrams or charts.

**Feature request:** I suggest introducing a brand-new function called "Virtual Pointer". The virtual pointer is a tool that permits presenters to highlight or use a virtual pointer with a laser to attract focus on specific areas of their presentation. 

 **Feature's Behaviour:** A keyboard command or a button on the toolbar of the presentation software can be used to launch the Virtual Pointer. Once turned on, the presenter could use their mouse or trackpad to move the pointer around the screen.

**Comment by Martinomagnifico** [comment]
See https://github.com/hakimel/reveal.js/wiki/Plugins,-Tools-and-Hardware for several 3rd party plugins that do just that :-)
